public/app/features/org/OrgProfile.tsx

```diff
@@ -15,19 +16,20 @@ interface FormDTO {
 
 const OrgProfile = ({ onSubmit, orgName }: Props) => {
   const canWriteOrg = contextSrv.hasPermission(AccessControlAction.OrgsWrite);
+  const initialFormModel: FormDTO = { orgName };
+
+  const { handleSubmit, register } = useForm<FormDTO>({ defaultValues: initialFormModel });
```

Comment:
```suggestion
const { handleSubmit, register } = useForm<FormDTO>({ defaultValues: { orgName } });
```
This can be simplified.